pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -329,7 +330,15 @@ func (self *RefsHelper) NewBranch(from string, fromFormattedName string, suggest
 	)
 
 	if suggestedBranchName == "" {
-		suggestedBranchName = self.c.UserConfig().Git.BranchPrefix
+		var err error
```

Comment:
Does this need to be declared separately?

Reply:
Yes, I think so. Without this you'd have to say `:=` below, but that makes `suggestedBranchName` a new variable in that scope, so that doesn't work.

Reply:
ah yes, makes sense